pyproject.toml

```
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]
+
+[project]
+name = "format-converters"
+version = "0.1"
```

```suggestion
version = "0.1.0"
```

I don't care about the actual version, but leaving out the `patch` segment seems wrong ;)